src/Microsoft.AspNetCore.Razor.Evolution/ITagHelperBoundAttributeDescriptorBuilder.cs

```diff
@@ -0,0 +1,260 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNetCore.Razor.Evolution
+{
+    public sealed class ITagHelperBoundAttributeDescriptorBuilder
```

Comment:
How about we compromise and add this to https://github.com/aspnet/Razor/issues/1006

Not worth blocking on